ccd-definition/AuthorisationCaseField.json

```diff
@@ -419,6 +433,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "children1",
```

Comment:
Maybe it doesnt matter but I prefer a name like newChildren. If this doesnt matter and is too big of a change then this is totally ok 😄

Reply:
The name is only temporary, i'll keep it 'children1' for now.

Reply:
👍